stackrox/base/policy/HorizontalPodAutoscaler.yaml

```diff
@@ -0,0 +1,23 @@
+# Source: stackrox-central-services/templates/02-scanner-08-hpa.yaml
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
```

Comment:
this should be in its own directory called `hotizontalpodautoscaler` as it is not a policy of any kind